compareBuckets/MongoOplogInterface.js

```diff
@@ -0,0 +1,156 @@
+/*
+  fake backend that uses MongoDB for oplog (manual debug)
+
+  - start mongodb (>=3.6)
+  $ git clone github.com/scality/Zenko-MD
+  $ cd Zenko-MD
+  $ MDP_DRIVER=mongodb npm start
+*/
+const MongoClient = require('mongodb').MongoClient;
+const bson = require('bson');
+const async = require('async');
+const Flusher = require('./Flusher');
+const { jsutil } = require('arsenal');
+const { isMasterKey } = require('arsenal/lib/versioning/Version');
+const werelogs = require('werelogs');
+
+werelogs.configure({
+    level: 'info',
+    dump: 'error',
+});
+
+const databaseName = 'metadata';
+
+class MongoOplogInterface {
+
+    constructor() {
+        this.logger = new werelogs.Logger('MongoOplogInterface');
+    }
+
+    start(bucketName, persist, persistData, stopAt, cb) {
```

Comment:
I really think we should split this function up and as stated before maybe switch to async/await. If not splitting use `function funcName` syntax over the anonymous array functions to provide context.